netlogo-gui/src/main/properties/Label.scala

```diff
@@ -0,0 +1,15 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.properties
+
+abstract class Label(accessor: PropertyAccessor[String], useTooltip: Boolean)
+  extends PropertyEditor(accessor, useTooltip)
+{
+  setLayout(new java.awt.BorderLayout(BORDER_PADDING, 0))
+  val label = new javax.swing.JLabel(accessor.displayName)
+  tooltipFont(label)
+  label.setFont(label.getFont.deriveFont(java.awt.Font.BOLD))
+  add(label, java.awt.BorderLayout.CENTER)
+  override def get: Option[String] = None
```

Comment:
the bug *Invalid value for Note: hover over underlined text for additional info about a field* can be fixed by changing this to
`
  override def get: Option[String] = { Some(accessor.displayName)}`